src/flamegpu/util/Environment.cpp

```diff
@@ -0,0 +1,56 @@
+#include "flamegpu/util/Environment.h"
```

Comment:
Might be worth renaming this and `Environment.h` so it's not to be confused with FLAMEGPU model environment.

SysEnv or something?

Reply:
Alternativeley remove all setting of temporary env vars, and it can be removed as it's then only needed to get the value of `FLAMEGPU_SHARE_USAGE_STATISTICS` in `Telemetry.cpp`

Reply:
File no longer exists.